lxd/network/acl/acl_ovn.go

```diff
@@ -1067,14 +1069,30 @@ func ovnParseLogEntry(input string, prefix string) string {
 	}
 
 	// Filter for our ACL.
-	if !strings.HasPrefix(aclEntry["name"], prefix) {
+	if prefix != "" && !strings.HasPrefix(aclEntry["name"], prefix) {
 		return ""
 	}
 
-	// Parse the timestamp.
-	logTime, err := time.Parse(time.RFC3339, fields[0])
-	if err != nil {
-		return ""
+	var logTime time.Time
+	var err error
+	// When reading from a log file, the timestamp is provided as a message parameter.
+	if timestamp == "" {
```

Comment:
Lets break the time parsing aspect of this into a separate function, and then have the timestamp always provided to this function as an time.Time  argument (reduce cyclomatic complexity) .